apps/wallets/smart-wallet/react/src/app/env.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+
+const envSchema = z.object({
+    REACT_APP_CROSSMINT_API_KEY_PROD: z.string(),
```

Comment:
I think I would make prod optional, as it will be harder to get such key and i think many folks will just want to test in staging